README.md

```diff
@@ -52,6 +52,8 @@ In Julia:
 
     # ENV["PYTHON"] = "/usr/bin/python3.7"           # example for *nix
     Pkg.build("PyCall")
+    
+Julia needs to be re-launched for this change to take effect.
```

Comment:
This is only true if you've already imported `PyCall` in the current session.

Reply:
Ok, I see. Then maybe leave it as it is.